python/felis/tap.py

```diff
@@ -133,7 +132,7 @@ class Tap11KeyColumns(Tap11Base):
     )
 
 
-class TapLoadingVisitor(Visitor[None, tuple, Tap11Base, None, tuple, None, None]):
+class TapLoadingVisitor:
```

Comment:
Maybe rename it to something else as it is not look like visitor pattern anymore, just a regular class that converts `datamodel` into TAP representation.

Reply:
It does the same thing, but not in a "visitor" way. The type of the visitor was generic because we had visitors that converted that schema into different types of objects. Visitor patter was probably not super-useful for that purpose, but this is how it was done initially.

Reply:
I am just going to leave this for now, as I only removed the complex type signature from the class and did not touch the control flow at all. I didn't change any of its functionality. But I take your point that we may eventually want to name it something else.